babel.config.js

```diff
@@ -475,15 +475,14 @@ function pluginNodeImportInteropBabel({ template }) {
 }
 
 function pluginNodeImportInteropRollup({ types: t }) {
-  const depsUsing__esModuleAndDefaultExport = [
-    src => src.startsWith("babel-plugin-polyfill-"),
-  ];
+  const depsUsing__esModuleAndDefaultExport = src =>
+    src.startsWith("babel-plugin-polyfill-") || src === "regenerator-transform";
```

Comment:
Just wondering, is there anything else other than `regenerator-transform` that would need to be added? Would we need to update this if we added another similar dep?

Oh so via https://github.com/babel/babel/pull/13017, it's only esm/cjs packages that have been transformed with babel?

Reply:
Yes, the only problem is dependencies that have been transformed with Babel.

I managed to create a working build just with this change so I _think_ it's the only one (also I tried looking for other `export ... from` of dependencies and this is the only one), but if we'll ever find a new one we can always add it to this list.

Reply:
Ok was just worried it would break without us knowing and whether one could verify!